pkg/models/host.go

```diff
@@ -3,10 +3,22 @@
 package models
 
 import (
+	"bytes"
+	"context"
 	"fmt"
+	"net"
+	"os"
+	"path/filepath"
 	"strings"
+	"time"
 
 	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/melbahja/goph"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	maxResponseSize = 102400
```

Comment:
I believe you changed the number by mistake. Could you add a comment on why you choose this number?

Reply:
this is just max size of byte slice allocated to the response to be parsed.
it can be any number that's why I made it bigger

Reply:
added comment